roles/custom/matrix-bridge-mautrix-wsproxy/templates/systemd/matrix-mautrix-wsproxy-syncproxy.service.j2

```diff
@@ -0,0 +1,43 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Mautrix wsproxy bridge
+{% for service in matrix_mautrix_wsproxy_syncproxy_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_mautrix_wsproxy_syncproxy_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ devture_systemd_docker_base_systemd_unit_home_path }}"
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} kill matrix-mautrix-wsproxy-syncproxy 2>/dev/null'
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} rm matrix-mautrix-wsproxy-syncproxy 2>/dev/null'
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre={{ matrix_host_command_sleep }} 5
+
+ExecStart={{ devture_systemd_docker_base_host_command_docker }} run --rm --name matrix-mautrix-wsproxy-syncproxy \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--network={{ matrix_docker_network }} \
+			-p {{ matrix_mautrix_wsproxy_syncproxy_port }}:29331 \
+			-e DATABASE_URL={{ matrix_mautrix_signal_wsproxy_syncproxy_connection_string }} \
+			-e HOMESERVER_URL={{ matrix_homeserver_container_url  }} \
+			-e SHARED_SECRET={{ matrix_mautrix_wsproxy_syncproxy_shared_secret }} \
```

Comment:
Creating an `env` file under `matrix_mautrix_wsproxy_base_path` with these variables is preferrable, as it keeps the systemd service file tidier, does not leak the secrets to people running `ps`, and allows for changing the environment variables without having to run `systemctl daemon-reload`.

See the `matrix-sliding-sync` role for an example.

----

Some roles usually create a `labels` file the same way they create the `env` file under the base path. This labels file would contain container labels to assist integration with Traefik. See the `matrix-client-element` role for an example.